### PR TITLE
[BPK-4415] Disabled chips are no longer pressable

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

**Review by olliecurtis — COMMENTED** [review]
LGTM! Once PR is deployed will check and approve!

File: packages/bpk-mixins/src/mixins/_chips.scss
Comment (georgegillams) [inline packages/bpk-mixins/src/mixins/_chips.scss:38]: Another way to achieve this would be to explicitly set the disabled background colour as disabled style takes precedence over active style.

I wonder if the cursor should be `not-allowed` for disabled ones?

Comment (shaundon) [reply packages/bpk-mixins/src/mixins/_chips.scss:38]: > I wonder if the cursor should be not-allowed for disabled ones?

I don't think we should do that – we let the browser decide in every other case so it'd be an unusual thing for us to do.

i don't think

**Review by olliecurtis — APPROVED** [review]

File: packages/bpk-mixins/src/mixins/_chips.scss
Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_chips.scss:38]: We literally do that for button

**Pull request merged**
